### DISCUSSION TITLE
Support for R

### DISCUSSION THREAD

**Discussion by sgavathe** [paketo-buildpacks/feedback#226] [Ideas]
It will be great, if there is a buildpack for R available through Paketo system. There is a one from Cloud Foundry, but it looks dated and not maintained. Initially if it supports package manager such as packrat, that would be sufficient.

**Comment by robdimsdale** [comment]
We're not averse to creating one, but it's not something we've received much feedback from the community yet.

**Comment by dmikusa** [comment]
I would also suggest trying your own hand at a buildpack. It may not end up being as tricky as you think. You can get a nice walk-through [here](https://buildpacks.io/docs/buildpack-author-guide/create-buildpack/). Even if it's just a proof of concept, that shows how you might go about an R buildpack, that's all very helpful.

For what it's worth, we also have https://github.com/paketo-community for incubating new buildpacks. If you get something together and want to share it. That's an option too.

Feel free to ask questions about developing buildpacks here or on the Paketo Slack as well.

**Comment by sgavathe** [comment]
Yeah managed to get the R buildpack fully working. Thank you @robdimsdale @dmikusa.
